lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
And I was just calling it `dropdown-button` (with the shorthand option). Seems in keeping with the others right?